packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -2276,6 +2276,59 @@ addEventListener('fetch', event => {});`
       `);
       expect(std.err).toMatchInlineSnapshot(`""`);
     });
+
+    it("should generate an asset manifest with keys relative to site.bucket", async () => {
+      const assets = [
+        { filePath: "file-1.txt", content: "Content of file-1" },
+        { filePath: "file-2.txt", content: "Content of file-2" },
+      ];
+      const kvNamespace = {
+        title: "__test-name-workers_sites_assets",
+        id: "__test-name-workers_sites_assets-id",
+      };
+
+      writeWranglerToml({
+        main: "./src/index.js",
+        site: {
+          bucket: "assets",
+        },
+      });
+      writeWorkerSource({ basePath: "src", type: "esm" });
+      writeAssets(assets);
+      mockUploadWorkerRequest({
+        expectedBindings: [
+          {
+            name: "__STATIC_CONTENT",
+            namespace_id: "__test-name-workers_sites_assets-id",
+            type: "kv_namespace",
+          },
+        ],
+        expectedModules: {
+          __STATIC_CONTENT_MANIFEST:
+            '{"file-1.txt":"file-1.2ca234f380.txt","file-2.txt":"file-2.5938485188.txt"}',
+        },
+      });
+      mockSubDomainRequest();
+      mockListKVNamespacesRequest(kvNamespace);
+      mockKeyListRequest(kvNamespace.id, []);
+      mockUploadAssetsToKVRequest(kvNamespace.id, assets);
+
+      process.chdir("./src");
+      await runWrangler("publish");
+      process.chdir("../");
```

Comment:
(feel free to answer this whenever you're back): does this second `process.chdir` do anything?

Reply:
Just undoes the chdir from 2 lines up, so it's back at "root". I do that mostly for cleanup/aesthetics haha, it's not strictly necessary for this test.